charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -49,7 +49,7 @@ insecureScheme:
   nginx:
     image:
       registry: docker.io
-      repository: nginx
+      repository: nginxinc/nginx-unprivileged
```

Comment:
Prefer to start using the chainguard images once, this one is resolved.

https://github.com/chainguard-images/images/issues/93

We use also the wait-for-it image from chainguard.

Reply:
Sounds good. Maybe file an issue for it so we don't forget about it?